packages/start-frontend/src/index.ts

```diff
@@ -326,6 +327,16 @@ async function run() {
     removeEslintConfig();
   }
 
+  const { default: nodePlop } = await import("node-plop");
```

Comment:
Normally importing `nodePlop` throws an `ERR_REQUIRE_ESM` error. (See https://github.com/monstar-lab-oss/ml-frontend/issues/1222#issuecomment-2089745652)

In order to successfully build with `node-plop`, dynamic importing it seem to work, with the `plopFile.ts` exporting using CommonJS syntax.